addons/auth_ldap/models/res_company_ldap.py

```diff
@@ -6,7 +6,7 @@
 from ldap.filter import filter_format
 
 from odoo import _, api, fields, models, tools
-from odoo.exceptions import AccessDenied
+from odoo.exceptions import AccessDenied, ValidationError
```

Comment:
unused import

Reply:
Fixed :+1: (I'd added a "hard" constraint on the presence of placeholders but after reflexion that seemed a bit too extreme, but I forgot to remove the import).